bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -0,0 +1,52 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/browsers/chromebook" %}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('the-latest-firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('quantum') }}
+{% endblock %}
+
+{% block content %}
+
+{% call hero(
+  title=ftl('the-latest-firefox'),
+  class='mzp-has-image mzp-t-product-firefox mzp-t-dark',
+  image_url='img/firefox/enterprise/fx-browser-img.svg',
+  include_highres_image=True,
+  include_cta=True,
+  heading_level=1
+) %}
+<p>{{ ftl('firefox-quantum-was') }}</p>
+<p><a href="https://blog.mozilla.org/blog/2017/11/14/introducing-firefox-quantum/">{{ ftl('learn-more-about') }}</a></p>
```

Comment:
Should this URL have utm params? It also seems a little strange to me that a link to this blog post is the only CTA (i.e. no download button, other than the one in the navigation)

Reply:
In the design https://github.com/mozilla/bedrock/issues/9501 there is a download button.